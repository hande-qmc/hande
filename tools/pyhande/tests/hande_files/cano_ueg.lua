-- Create output with:
-- $[HANDE DIR]/bin/hande.x cano_ueg.lua > cano_ueg.out 2> cano_ueg.err
-- Note that these settings are just for testing...
sys = ueg {
    dim = 3,
    nel = 14,
    ms = 0,
    cutoff = 1,
}

canonical_estimates {
    sys = sys,
    canonical_estimates = {
        ncycles = 2,
        nattempts = 10000,
        beta = 0.2,
        rng_seed = 748
    },
}
