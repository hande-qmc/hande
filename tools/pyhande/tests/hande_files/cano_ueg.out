
   HANDE

 ================================================================
Version information
-------------------
Version       | v1.4-25-g4efb802
Commit hash   | 4efb80270c639ebaf202fe7b326e94a9ccfefafe
Commit author | Verena Neufeld
Commit date   | Thu Mar 21 12:08:44 2019 +0000
Branch        | master

Configuration and build information
-----------------------------------
Who compiled             | verena
Compilation hostname     | verena-Think
Operating system         | Linux-4.4.0-176-generic
CMake version            | 3.10.2
CMake generator          | Unix Makefiles
Build type               | release
Configuration time       | 2020-03-31 15:44:09 [UTC]
Python version           | 3.6.10
Fortran compiler         | /usr/bin/gfortran
C compiler               | /usr/bin/gcc
DET_SIZE set to          | 64
POP_SIZE set to          | 64
dSFMT Mersenne exponent  | 19937
MPI parallelization      | OFF
OpenMP parallelization   | OFF
Lua version              | 5.3.5
HDF5 library version     | 1.10.2

Further components
------------------
DISABLE_HDF5 not defined.  HDF5 enabled.
DISABLE_LANCZOS defined.  Lanczos disabled.
DISABLE_UUID not defined.  UUID enabled.
PARALLEL not defined.  MPI parallelization disabled.
DISABLE_SCALAPACK not defined.  ScaLAPACK enabled.
SINGLE_PRECISION not defined.  Double precision used throughout.
USE_POPCNT not defined. Internal POPCNT procedure used.

Runtime information
-------------------
Hostname:
    verena-Think
Current working dir:
    /home/verena/code3/hande/tools/pyhande/pyhande/tests/hande_files
Started running on 01/04/2020 at 21:02:50
 Calculation UUID: 98db19fc-8ab2-4f5a-82d0-5c4c10340a1c.
 ================================================================

 Input options
 -------------

-- Create output with:
-- $[HANDE DIR]/bin/hande.x cano_ueg.lua > cano_ueg.out 2> cano_ueg.err
-- Note that these settings are just for testing...
sys = ueg {
    dim = 3,
    nel = 14,
    ms = 0,
    cutoff = 1,
}

canonical_estimates {
    sys = sys,
    canonical_estimates = {
        ncycles = 2,
        nattempts = 10000,
        beta = 0.2,
        rng_seed = 748
    },
}

 -------------

 Basis functions
 ---------------

 Spin given in units of 1/2.
 k-points given in terms of the reciprocal lattice vectors of the crystal cell.

 index   k-point            ms       <i|h|i>
     1   (  0,  0,  0)      1        0.00000000E+00
     2   (  0,  0,  0)     -1        0.00000000E+00
     3   (  0,  0, -1)      1        1.30773168E+00
     4   (  0,  0, -1)     -1        1.30773168E+00
     5   (  0, -1,  0)      1        1.30773168E+00
     6   (  0, -1,  0)     -1        1.30773168E+00
     7   ( -1,  0,  0)      1        1.30773168E+00
     8   ( -1,  0,  0)     -1        1.30773168E+00
     9   (  1,  0,  0)      1        1.30773168E+00
    10   (  1,  0,  0)     -1        1.30773168E+00
    11   (  0,  1,  0)      1        1.30773168E+00
    12   (  0,  1,  0)     -1        1.30773168E+00
    13   (  0,  0,  1)      1        1.30773168E+00
    14   (  0,  0,  1)     -1        1.30773168E+00
    15   (  0, -1, -1)      1        2.61546336E+00
    16   (  0, -1, -1)     -1        2.61546336E+00
    17   ( -1,  0, -1)      1        2.61546336E+00
    18   ( -1,  0, -1)     -1        2.61546336E+00
    19   (  1,  0, -1)      1        2.61546336E+00
    20   (  1,  0, -1)     -1        2.61546336E+00
    21   (  0,  1, -1)      1        2.61546336E+00
    22   (  0,  1, -1)     -1        2.61546336E+00
    23   ( -1, -1,  0)      1        2.61546336E+00
    24   ( -1, -1,  0)     -1        2.61546336E+00
    25   (  1, -1,  0)      1        2.61546336E+00
    26   (  1, -1,  0)     -1        2.61546336E+00
    27   ( -1,  1,  0)      1        2.61546336E+00
    28   ( -1,  1,  0)     -1        2.61546336E+00
    29   (  1,  1,  0)      1        2.61546336E+00
    30   (  1,  1,  0)     -1        2.61546336E+00
    31   (  0, -1,  1)      1        2.61546336E+00
    32   (  0, -1,  1)     -1        2.61546336E+00
    33   ( -1,  0,  1)      1        2.61546336E+00
    34   ( -1,  0,  1)     -1        2.61546336E+00
    35   (  1,  0,  1)      1        2.61546336E+00
    36   (  1,  0,  1)     -1        2.61546336E+00
    37   (  0,  1,  1)      1        2.61546336E+00
    38   (  0,  1,  1)     -1        2.61546336E+00

 Number of electrons: 14
 Number of basis functions: 38

 Bit-length of integers used to store determinant bit-strings: 64
 Number of integers used to store determinant bit-strings: 1

 Canonical energy
 ----------------

 -- Start JSON block --
 {
     "system": {
         "nbasis": 38,   
         "nel": 14,   
         "nvirt": 24,   
         "Ms": 0,   
         "nalpha": 7,   
         "nbeta": 7,   
         "nvirt_alpha": 12,   
         "nvirt_beta": 12,   
         "nsym": 19,   
         "sym0": 1,   
         "sym_max": 19,   
         "nsym_tot": 19,   
         "sym0_tot": 1,   
         "sym_max_tot": 19,   
         "symmetry": 2147483647,   
         "tot_sym": false,   
         "aufbau_sym": true,   
         "max_number_excitations": 14,   
         "ueg": {
             "r_s": 1.00000000,   
             "ecutoff": 1.00000000,   
             "k_fermi": 1.91915829,   
             "E_fermi": 1.84158428,   
             "ktwist": [0.00000000,0.00000000,0.00000000],
             "L": [3.88512994,3.88512994,3.88512994]
         }
     },
     "all_spin_sectors": false,   
     "beta": 0.20000000,   
     "fermi_temperature": false,   
     "nattempts": 10000,   
     "free_energy_corr": -96.66701818,   
     "ncycles": 2,   
     "chem_pot": -0.64446167,   
     "rng_seed": 748    
 }
 -- End JSON block --


 Reference determinant and so symmetry sector selected using the Aufbau principle.
 Selected symmetry  1.

 <T>_0: Estimate for kinetic energy in non-interacting ensemble i.e. 1/Z_0 Tr(\rho_0 T).
 <V>_0: Estimate for potential energy in non-interacting ensemble energy i.e. 1/Z_0 Tr(\rho_0 V).
 Tr(T\rho_HF): Estimate for numerator of "Hartree-Fock" kinetic energy.
 Tr(V\rho_HF): Estimate for numerator of "Hartree-Fock" potential energy.
 Tr(\rho_HF): Estimate for denominator of "Hatree-Fock" energy i.e. Z_HF.
 N_ACC/N_ATT: Ratio of number of generated N particle states to number of attempts. Also Estimate for Z_GC(N)/Z_GC.

 # iterations                 <T>_0                 <V>_0              Tr(T\rho_HF)          Tr(V\rho_HF)           Tr(\rho_HF)           N_ACC/N_ATT
            1      2.7943370015E+01     -1.5740506968E+00          2.5170736424E+01     -1.4249954801E+00      9.0253160272E-01      3.4800000000E-02
            2      2.7726172803E+01     -1.5817153286E+00          2.5010826838E+01     -1.4341367788E+00      9.0391912264E-01      3.4700000000E-02

 Timing breakdown
 ----------------

 Time for each calculation section (seconds):

 UEG initialisation         : .00
 Canonical energy estimation: .00

 ================================================================
 Finished running on 01/04/2020 at 21:02:50
 Wall time (seconds):                           0.00
 CPU time (per processor, seconds):             0.00
 ================================================================

